Tasks/Xcode/xcode.ts

```diff
@@ -85,6 +85,29 @@ async function run() {
             }
         }
 
+        let destinations: string[];
+        let platform: string = tl.getInput('platform', false);
+        if (platform === 'macOS') {
+            destinations = ['platform=macOS'];
+        }
+        else if (platform && platform !== 'default') {
+            // To be yaml friendly, destinationType is optional and we default to simulators.
+            let destinationType: string = tl.getInput('destinationType', false);
+
+            let devices: string[];
+            let targetingSimulators: boolean;
+            if (destinationType !== 'devices') {
```

Comment:
why not a positive check here? if (destinationType === 'simulator') and one for 'devices'

Reply:
I want yaml users to just be able to say:
 -platform: iOS
 -simulators: iPhone X

Without supplying destinationType. However I rewritten this code per Luke's suggestion.